orchagent/aclorch.cpp

```diff
@@ -3165,7 +3183,17 @@ bool AclOrch::addAclRule(shared_ptr<AclRule> newRule, string table_id)
         return false;
     }
 
-    return m_AclTables[table_oid].add(newRule);
+    if (!m_AclTables[table_oid].add(newRule))
```

Comment:
@stepanblyschak do we need to add `if (!newRule)` prior to this?

Reply:
We do, currently it is undefined behaviour if we pass nullptr regardless of this change. I will try to push it in another PR.